pyfstat/make_sfts.py

```diff
@@ -99,7 +99,8 @@ def __init__(
         sqrtSX=None,
         noiseSFTs=None,
         SFTWindowType=None,
-        SFTWindowBeta=0.0,
+        SFTWindowParam=0.0,
```

Comment:
Is this consistent with LALPulsar's default?

Reply:
This was a good catch indeed: MFD throws `Inconsistent Window-option: 'tukey' requires parameter = yes, but user supplied parameter = no` if trying to rely on the default (i.e. it's detecting "user did not make any conscious choice"). But as I had it implemented, we produced `window:      tukey(0)`.

Unfortunately going with `None` as our default triggers a rather ugly lal error `XLAL Error - XLALParseStringValueAsREAL8 (UserInputParse.c:168): strtod(): trailing garbage 'None' found after double-conversion of 'None'` so I now put in an explicit check via [XLALCheckNamedWindow](https://lscsoft.docs.ligo.org/lalsuite/lal/group___window__h.html#gaac5218a54390c9f427f8d359b598e9ec).

(954ddb8)